pipx/pipxrc.py

```diff
@@ -0,0 +1,196 @@
+import json
+from pathlib import Path
+from typing import List, Dict, NamedTuple, Any, Optional, TypeVar
+
+from pipx.Venv import PipxVenvMetadata
+
+
+PIPX_INFO_FILENAME = "pipxrc.json"
+
+
+class JsonEncoderPipx(json.JSONEncoder):
```

Comment:
Fine.  This started out doing more than just Paths, and evolved to only fixing the Paths (since Path's were the only thing distributed through the hierarchy and harder to fix by other means)

Reply:
Done. Using JsonEncoderHandlesPath